src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -5,13 +5,12 @@ import * as Schema from "Analytics/Schema"
 import FollowArtistButton from "Components/FollowButton/FollowArtistButton"
 import React, { Component, Fragment } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
-import { AppState } from "Router/state"
+import { ContextConsumer } from "Router/Artsy2"
```

Comment:
@alloy - my idea is to adjust reactions folder structure a bit so that things will look like 

```
/Artsy
  /Analytics
    ...
  /Router
    ...
  ArtsyContext.tsx
  index.tsx
```
Thoughts? It seems like placing this file under `/Components` might be a little less intuitive at this point in time.